packages/structure/src/model/RWTOML.ts

```diff
@@ -26,6 +26,7 @@ export class RWTOML extends FileNode {
   }
   *diagnostics() {
     try {
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
       this.parsedTOML
```

Comment:
Aside: this is part of why many people don't like getters 😄. The fact that an object access can throw is confusing and surprising.

Reply:
Haha, yeah, I can see that!